aptos-move/framework/aptos-experimental/doc/confidential_asset.md

```diff
@@ -76,14 +76,14 @@ It enables private transfers by obfuscating token amounts while keeping sender a
 <b>use</b> <a href="../../aptos-framework/doc/chain_id.md#0x1_chain_id">0x1::chain_id</a>;
 <b>use</b> <a href="../../aptos-framework/doc/coin.md#0x1_coin">0x1::coin</a>;
 <b>use</b> <a href="../../aptos-framework/doc/dispatchable_fungible_asset.md#0x1_dispatchable_fungible_asset">0x1::dispatchable_fungible_asset</a>;
-<b>use</b> <a href="../../aptos-framework/../aptos-stdlib/../move-stdlib/doc/error.md#0x1_error">0x1::error</a>;
```

Comment:
Why do we see all those diffs in the generated doc -- and are they correct? Wouldn't expect an optimization to change output of docgen.

Reply:
Maybe we need to add an option to separately compile package just for the document to avoid this issue?

Reply:
Updated the PR to generate the doc with no experiment flags. However, it increases the compilation time.